### PR TITLE
Hotfix/header

### PR DESCRIPTION
**Pull request by RomanRymarchuk1**

### CODE REVIEW

File: client/src/components/TourInfoDialog/TourInfoDialog.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/forbid-prop-types */
```
Comment (Ahmed-Fahmy-Dev) [inline client/src/components/TourInfoDialog/TourInfoDialog.js:1]: this is already included in the eslint config file as a general rule.   you can remove this line

**Review by Ahmed-Fahmy-Dev — COMMENTED** [review]
please add this line to the Header.test.js and then update your snapthot and this should solve your issue.  :

`jest.mock('@mui/material/Slide', () => ({ children }) => <div>{children}</div>);
`

File: client/src/layout/Header/Header.js [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+  // TODO: fix test warn;
+  return (
+    <Slide appear={false} direction="down" in={!trigger}>
+      {children}
+    </Slide>
+  );
+};
 
 const Header = () => {
   const [scrollY, setScrollY] = useState();
```
Comment (Ahmed-Fahmy-Dev) [inline client/src/layout/Header/Header.js:18]: i dont think this is needed anymore

**Review by Ahmed-Fahmy-Dev — COMMENTED** [review]
also the useEffect event listener i think it is not needed anymore

File: client/src/layout/Header/Header.js (hunk 2 above)
Comment (RomanRymarchuk1) [reply client/src/layout/Header/Header.js:18]: it also needed for boxShadow

File: client/src/components/TourInfoDialog/TourInfoDialog.js (hunk 1 above)
Comment (RomanRymarchuk1) [reply client/src/components/TourInfoDialog/TourInfoDialog.js:1]: i don’t  see it in eslint config

**Review by Ahmed-Fahmy-Dev — APPROVED** [review]

**Review by valeriaholdovych — APPROVED** [review]

**Pull request merged**